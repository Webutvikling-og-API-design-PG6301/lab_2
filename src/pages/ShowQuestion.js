import React from "react";

const ShowQuestion = () => {
  return (
    <div>
      <h1>This is show question component</h1>
    </div>
  );
};

export default ShowQuestion;
