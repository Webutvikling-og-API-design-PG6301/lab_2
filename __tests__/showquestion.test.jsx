import React from "react";
import { render } from "react-dom";
import ShowQuestion from "../src/pages/ShowQuestion";

describe("show question page", () => {
  it("should give me some content", () => {
    const element = document.createElement("div");
    render(<ShowQuestion />, element);
    expect(element.innerHTML).toMatchSnapshot();
  });
});
