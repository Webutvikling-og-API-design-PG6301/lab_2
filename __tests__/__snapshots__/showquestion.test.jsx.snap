// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show question page should give me some content 1`] = `"<div><h1>This is show question component</h1></div>"`;
